rusoto/core/src/signature.rs

```diff
@@ -25,16 +25,27 @@ use credential::AwsCredentials;
 /// the Amazon Signature Version 4 signing process
 #[derive(Debug)]
 pub struct SignedRequest {
+    /// HTTP Method
```

Comment:
I'd prepend `The` to all of these. So like: `The AWS Region`. `The HTTP Method.`